README.md

```diff
@@ -1,14 +1,14 @@
 [![CircleCI](https://circleci.com/gh/SAP/yeoman-ui.svg?style=svg)](https://circleci.com/gh/SAP/yeoman-ui)
 ![GitHub license](https://img.shields.io/badge/license-Apache_2.0-blue.svg)
 
-# Yeoman UI
+# Application Wizard
 
 ![](screenshot.png)
 
 ## Description
 Provide rich user experience for Yeoman generators using VSCode extension or the browser.
 The repository contains three main packages:
-* **Frontend** - The Yeoman UI as a standalone vue.js application.
+* **Frontend** - The Application Wizard as a standalone vue.js application.
 * **Backend** - The backend part which communicate with Yeoman and the system. Runs as a VSCode extension or node.js application.
 * **Yeoman example generator** - Example generator to show usages and test the platform.
```

Comment:
@slavik-lvovsky  I would prefer this wording:

> Provides a rich user experience for Yeoman generators. Runs in a web browser or in Visual Studio Code.
The repository contains three main packages:
> * **Frontend** (`frontend`) - A standalone vue.js application
> * **Backend** (`backend`) - Serves the frontend and runs Yeoman generators. It can run as a Visual Studio Code extension or as a web/WebSocket server.
> * **Sample Yeoman generator** (`generator-foodq`) - A sample Yeoman generator that shows usage and capabilities.

But, please consult with Paula.